frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -59,6 +60,72 @@ export const validationSchema = yup.object().shape({
       .string()
       .matches(pathRegex, { message: 'Path must start with /.', excludeEmptyString: true }),
   }),
+  limits: yup.object().shape({
+    cpu: yup.object().shape({
+      request: yup
+        .number()
+        .integer('Request must be an Integer.')
```

Comment:
I believe values like 1.5Gi are allowed. We should verify, though.

Reply:
remove this integer validation.